Rsk.Samples.IdentityServer.AdminUiIntegration/Rsk.Samples.IdentityServer.AdminUiIntegration.csproj

```diff
@@ -32,5 +32,8 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="6.0.16" />
     <PackageReference Include="Pomelo.EntityFrameworkCore.MySql" Version="6.0.2" />
     <PackageReference Include="System.Drawing.Common" Version="4.7.3" />
+    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="6.30.1" />
+	<PackageReference Include="Rsk.Saml.DuendeIdentityServer" Version="7.0.3" />
```

Comment:
nit pick, indentation

Reply:
It shows correctly when loading it locally